src/shared/containers/GetOrganizations.tsx

```diff
@@ -35,108 +35,89 @@ import {convertStringToEpoch} from 'src/shared/utils/dateTimeUtils'
-  const quartzMeStatus = useSelector(getQuartzMeStatus)
-  const quartzMe = useSelector(getQuartzMe)
-  const quartzIdentityStatus = useSelector(selectQuartzIdentityStatus)
+  const {account, user, org: quartzOrg} = identity.currentIdentity
+  const identityStatus = useSelector(selectQuartzIdentityStatus)
   const {id: meId = '', name: email = ''} = useSelector(getMe)
-  const {account} = identity.currentIdentity
 
   const dispatch = useDispatch()
 
   // This doesn't require another API call.
   useEffect(() => {
-    if (status === RemoteDataState.NotStarted) {
+    if (orgLoadingStatus === RemoteDataState.NotStarted) {
       dispatch(getOrganizations())
     }
-  }, [dispatch, status]) // eslint-disable-line react-hooks/exhaustive-deps
+  }, [dispatch, orgLoadingStatus])
 
   useEffect(() => {
     if (
       CLOUD &&
-      email &&
-      org?.id &&
-      account?.id &&
-      account?.name &&
+      user.email &&
+      quartzOrg.id &&
+      account.id &&
+      account.name &&
       isFlagEnabled('rudderstackReporting')
     ) {
       identify(meId, {
-        email,
-        orgID: org.id,
+        email: user.email,
+        orgID: quartzOrg.id,
         accountID: account.id,
         accountName: account.name,
       })
     }
-  }, [meId, email, org?.id, account?.id, account?.name])
+  }, [meId, user.email, quartzOrg.id, account.id, account.name])
 
   useEffect(() => {
-    if (
-      CLOUD &&
-      quartzMeStatus === RemoteDataState.NotStarted &&
-      quartzIdentityStatus === RemoteDataState.NotStarted
-    ) {
+    if (CLOUD && identityStatus === RemoteDataState.NotStarted) {
       dispatch(getQuartzIdentityThunk())
     }
-  }, [quartzMeStatus, quartzIdentityStatus]) // eslint-disable-line react-hooks/exhaustive-deps
+  }, [dispatch, identityStatus])
 
   // This doesn't require another API call.
   useEffect(() => {
     if (
       isFlagEnabled('credit250Experiment') &&
-      quartzMeStatus === RemoteDataState.Done &&
-      status === RemoteDataState.Done
+      identityStatus === RemoteDataState.Done &&
+      orgLoadingStatus === RemoteDataState.Done
     ) {
-      const {
-        accountType: account_type,
-        accountCreatedAt: account_created_at = '',
-      } = quartzMe
       const orgId = org?.id ?? ''
       window.dataLayer = window.dataLayer ?? []
       window.dataLayer.push({
         identity: {
-          account_type,
-          account_created_at: convertStringToEpoch(account_created_at),
+          account_type: account.type,
+          account_created_at: convertStringToEpoch(account.accountCreatedAt),
           id: meId,
           email,
           organization_id: orgId,
         },
       })
     }
-  }, [quartzMeStatus, status]) // eslint-disable-line react-hooks/exhaustive-deps
-
-  const isRegionBeta = quartzMe?.isRegionBeta ?? false
-  const accountType = quartzMe?.accountType ?? 'free'
+  }, [identityStatus, orgLoadingStatus]) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
We could add `account.type`, `meId`, etc. to the dependency array here, but this looks _super_ intentional given that the `useEffect` hook is keyed off of the redux loading state and not directly off of the properties being sent to google analytics, so I'd rather not mess with the array at this time.